css/src/components/dropdown.css

```diff
@@ -19,6 +19,7 @@
 .Dropdown-input,
 .Dropdown-input:focus-within {
   border: unset !important;
+  box-shadow: none !important;
 }
```

Comment:
Please handle all the 3 states of search input "default", "focused", & "after search". Please refer to this link https://www.figma.com/file/w8sqBtJpvq86D06UE7gN0T/MDS---Web?node-id=2748%3A30842